mlocal/frags/build_runtime.mk

```diff
@@ -140,3 +140,13 @@ $(global_keyring_INSTALL): $(global_keyring)
 
 INSTALLFILES += $(global_keyring_INSTALL)
 
+# dmtcp  config file
+dmtcp_conf := $(SOURCEDIR)/etc/dmtcp-conf.yaml
```

Comment:
This file need to be added in https://github.com/apptainer/apptainer/blob/main/dist/nfpm/nfpm.yaml

Reply:
Rebased to incorporate nfpm changes and included in https://github.com/apptainer/apptainer/pull/109/commits/4bb73d153f5f9d75a1751f2d19ed3d48f0fedbe0